src/index.js

```diff
@@ -7,11 +7,17 @@ import { ToastContextProvider } from './Context/toast-context';
 import { UserLoginContextProvider } from './Context/user-login-context'
 import { TrendingVideosProvider } from './Context/trending-videos-context';
 import { AllVideosProvider } from './Context/all-videos-context'
+import { WatchLaterContextProvider } from './Context/watch-later-context'
+import { LikedVideosContextProvider } from './Context/liked-videos-context'
+import { DislikedVideosContextProvider } from './Context/disliked-videos-context'
 
 export { useToast } from './Context/toast-context';
 export { useUserLogin } from './Context/user-login-context'
 export { useTrendingVideos } from './Context/trending-videos-context'
 export { useAllVideos } from './Context/all-videos-context'
+export { useWatchLater } from './Context/watch-later-context'
+export { useLikedVideos } from './Context/liked-videos-context'
+export { useDislikedVideos } from './Context/disliked-videos-context'
 
 export { Navbar } from "./Components/Navbar/Navbar"
```

Comment:
Suggestion: you can make an index.js file in components and import all the components in one line and same for pages and contexts

Reply:
Instead of making 3 index.js file for components, context and pages, I thought to export them from 1 single index.js. I will read more about this. 😅